src/parser/csql_grammar.y

```diff
@@ -24133,25 +24135,85 @@ dblink_column_definition
 
 opt_access_modifier
     : /*empty*/
-      {{
+      {{ DBG_TRACE_GRAMMAR(opt_access_modifier, : );
 
 	$$ = PT_PRIVATE;
 
       DBG_PRINT}}
     | PRIVATE
-      {{
+      {{ DBG_TRACE_GRAMMAR(opt_access_modifier, | PRIVATE);
 
 	$$ = PT_PRIVATE;
 
       DBG_PRINT}}
     | PUBLIC
-      {{
+      {{ DBG_TRACE_GRAMMAR(opt_access_modifier, | PUBLIC);
 
 	$$ = PT_PUBLIC;
 
       DBG_PRINT}}
     ;
 
+/*
+ * To prevent the change of the error message below,
+ * the rules of opt_or_replace and opt_access_modifier are combined into opt_create_synonym and used.
+ * 
+ * Query: CREATE PARTITION TABLE;
+ * - AS-IS: Syntax error: unexpected 'partition', expecting CLASS or TABLE
+ * - TO-BE: Syntax error: unexpected 'partition', expecting SYNONYM
```

Comment:
Isn't AS-IS and TO-BE changed?

Reply:
There seems to be some confusion in the comments, so I'll replace AS-IS and TO-BE.